doc/data/glossary.yml

```diff
@@ -12,223 +12,229 @@
 #    - <Alt term 1>
 #    - <Alt term 2>
 
-Gateway:
-  description: Gateways form the bridge between devices and Network Servers. Devices use low power networks like LoRaWAN to connect to the Gateway, while the Gateway uses high bandwidth networks like WiFi, Ethernet or Cellular to connect to a Network Server.
+- term: Activation Mode
+  description: "**Over-the-Air Activation (OTAA)** is the preferred and most secure way to connect a device. Devices perform a join-procedure with the network, during which a dynamic Device Address is assigned and security keys are negotiated with the device. **Activation by Personalization (ABP)** requires hardcoding the Device Address as well as the security keys in the device. This strategy might seem simpler, because you skip the join procedure, but it has downsides related to security. **Multicast** is a virtual group of ABP devices which allows all devices to receive the same downlinks. Multicast groups do not support uplinks."
```

Comment:
Maybe also good to note that ABP devices can't switch to a different network provider